### PR TITLE
Feature/knex

### PR DESCRIPTION
**Pull request by qNanna**

### CODE REVIEW

File: api/controllers/apiConfigController.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import fs from 'fs'
+
+class ApiConfigController {
+  constructor (data) {
+    this.data = data
+  } // not used yet
+
+  async getDependencies (req, res) {
+    fs.readFile('./package.json', (err, data) => {
```
Comment (OlehSych) [inline api/controllers/apiConfigController.js]: why don't you use 'fs/promises' readFile? No need to use callbacks

File: api/controllers/orderController.js [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
-  async estimateOrder (req, res) {
-    const result = await orderService.glovoApiSend(`${process.env.GLOVO_API_DOMAIN}b2b/orders/estimate`, req.method, req.body)
-    result.total.discount = { discount: '20%', amount: orderService.getDiscont(result.total.amount) }
-    console.log(result)
-    res.json({ result })
+  async estimate (req, res) {
+    const key = utils.getHash(req.body)
+    let data = await redis.get(key)
+    if (!data) {
+      const locationArr = Object.entries(req.body).map(async ([key, el], index) => {
```
Comment (OlehSych) [inline api/controllers/orderController.js]: You need to expect only { from: string, to: string } request body structure.
So don't need to get location for all req.body, there can be some junk data, which will broke your app.
Example: if I send the req with body { from: 'bla-bla', to: 'bla-bla', destination: 'bla-bla', point: 'bla-bla' }
So your code will make an additional requests to location api with 'destination' and 'point' values.

File: api/controllers/apiConfigController.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import fs from 'fs'
+
+class ApiConfigController {
```
Comment (OlehSych) [inline api/controllers/apiConfigController.js:3]: maybe it is better to rename to serviceRouter, what do you think about that?
Bcz it handle request of getting service dependencies, so it's not connected to api config, just to service dependencies

File: services/locationLqService.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      return result
+    } catch (err) {
+      console.error(chalk.red(err))
+    }
+  }
+
+  async sendRequest (el) {
+    const newUrl = config.locationLqAPIUrl.replace('*KEY', config.locationLqAPIKey).replace('*ADRESS', el)
+    const data = await fetch(newUrl)
+    return (await data.json())[0] // [0] because api return all establishments of this address, take first
```
Comment (OlehSych) [inline services/locationLqService.js]: if api returns all matched addresses, so you don't need to fetch only first occurrence in 'sendRequest'.
It should be handled in 'getLocation' function

File: api/controllers/apiConfigController.js (hunk 1 above)
Comment (OlehSych) [reply api/controllers/apiConfigController.js]: I've sent you the example how to read file with promises & you added this example to utils, so just use the function from there.

File: api/controllers/apiConfigController.js (hunk 3 above)
Comment (qNanna) [reply api/controllers/apiConfigController.js:3]: but i already have serviceRouter named as apiConfigRouter, maybe i just rename apiConfigRouter and handle all from getDependencies function in lambda function  
"apiConfigRouter.get('/getDependencies', (req, res) => { HERE })"

**Pull request merged**